scalableminds/webknossos
# Add option to hide tree edges

This PR makes it possible for users to hide the edges of tree and also persist this setting.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a tracing
- Create two trees
- Enable/Disable the edge visibility of the trees using the context menu in the skeleton tab
- The nodes should always be visible (unless the whole tree is turned invisible)

### TODOs:
- [x] Adapt Tracing Store to changes. I need your help here @fm3 or maybe @frcroth 
- ~~[ ] Maybe also adapt wklibs client (as the trees have a new property)~~ decided to defer that
- ~~[ ] Adapt NML reading & writing~~ decided to defer that

### Issues:
- fixes [#7089](https://github.com/scalableminds/webknossos/issues/7089)

------
(Please delete unneeded items, merge only when none are left open)
